### PR TITLE
Login and Registration Form

### PR DESCRIPTION
**Pull request by nikhilkotiya**
## Related Issuse

- Info about Issue or bug

Closes: #[issue number that will be closed through this PR]

#### Describe the changes you've made

A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Review by khannakshat7 — COMMENTED** [review]
Please do this change

File: Elecktra/urls.py [hunk 1]
```diff
@@ -29,7 +29,8 @@
     path('announcements/',views.annnouncements,name="announcements"),
     path('feedback/',views.feedback,name="feedback"),
     path('logout/',views.logoutuser,name="logout"),
-    path('coordinates/',views.getmapcoordinates,name="coordinates"),
+    path('coordinates/',views.getmapcoordinates,name="coordinates")
```
[the remark this answers is not in the document]
Comment (nikhilkotiya) [reply Elecktra/urls.py]: Done.

**Comment by khannakshat7** [comment]
@nikhilkotiya Awesome work!! But few bugs- 
1. Logout button redirecting to the wrong URL.
2. Display First Name and Last Name before logout button when user is logged in. (Example: Welcome, Akshat Khanna)
3. After Signup redirect to log in page.
4. When password is incorrect show login page with error.

**Comment by khannakshat7** [comment]
This is the best contribution till now on this project. Great work buddy🙌🙌

**Comment by nikhilkotiya** [comment]
> This is the best contribution till now on this project. Great work buddy🙌🙌

Thank you sir

**Comment by khannakshat7** [comment]
Just do the required changes and I'll merge the PR

**Comment by khannakshat7** [comment]
Who created the Frontend for it?

**Comment by nikhilkotiya** [comment]
> @nikhilkotiya Awesome work!! But few bugs-
> 
> 1. Logout button redirecting to the wrong URL.
> 2. Display First Name and Last Name before logout button when user is logged in. (Example: Welcome, Akshat Khanna)
> 3. After Signup redirect to log in page.
> 4. When password is incorrect show login page with error.

Sir I can't find bug to displayay First Name and Last Name . 
Other bug removed

**Comment by nikhilkotiya** [comment]
> Who created the Frontend for it?

@sakshi

**Comment by nikhilkotiya** [comment]
> @nikhilkotiya Awesome work!! But few bugs-
> 
> 1. Logout button redirecting to the wrong URL.
> 2. Display First Name and Last Name before logout button when user is logged in. (Example: Welcome, Akshat Khanna)
> 3. After Signup redirect to log in page.
> 4. When password is incorrect show login page with error.

Sir bug 2 is in the base.html file but I did not commit this file. what did I do?

**Comment by khannakshat7** [comment]
> > Who created the Frontend for it?
> 
> @sakshi

Where is the PR for Front end

**Comment by khannakshat7** [comment]
> > @nikhilkotiya Awesome work!! But few bugs-
> > 
> > 1. Logout button redirecting to the wrong URL.
> > 2. Display First Name and Last Name before logout button when user is logged in. (Example: Welcome, Akshat Khanna)
> > 3. After Signup redirect to log in page.
> > 4. When password is incorrect show login page with error.
> 
> Sir bug 2 is in the base.html file but I did not commit this file. what did I do?
> Error is in line 64 ( user to user.first_name)

Can you can do that in base.html?

**Comment by nikhilkotiya** [comment]
@khannakshat7  sir can you please merge this branch because I can't take the latest code from master. There is an error when I do git pull origin master(maybe because I forked your project). I make a different Pull of base.html file bug.

**Comment by nikhilkotiya** [comment]
> > > Who created the Frontend for it?
> > 
> > 
> > @sakshi
> 
> Where is the PR for Front end

She is not answering

**Comment by khannakshat7** [comment]
@nikhilkotiya Awesome contribution dude🙌

**Comment by khannakshat7** [comment]
Don't forget to star⭐ the repo and follow me on GitHub

**Comment by nikhilkotiya** [comment]
> @nikhilkotiya Awesome contribution dude🙌

Sry sir due to lack of knowledge in git my work slow

**Comment by khannakshat7** [comment]
@nikhilkotiya That's how to learn. No issues keep going!!

**Comment by nikhilkotiya** [comment]
Screenshot (91)
Sir my master branch is now up to date and I just commit base.html file but why all files comes when i pull

**Pull request merged**